resolver/SuggestPackage.cc

```diff
@@ -304,6 +314,23 @@ bool SuggestPackage::tryPackageCorrections(core::Context ctx, core::ErrorBuilder
             for (auto match : missingExports) {
                 pkgCtx.addMissingExportSuggestions(e, match);
             }
+
+            if (auto fullConstant = ctx.locAt(unresolved.loc).source(ctx)) {
+                if (missingExports.size() == 1) {
+                    // if we have a single unambiguous export
+                    // suggestion, then we can use that information to
+                    // make the error message reference the exact
+                    // package we believe the constant came from
+                    auto missingExport = missingExports.front();
+                    auto pkgName = ctx.state.packageDB().getPackageInfo(missingExport.srcPkg).show(ctx);
+                    e.setHeader("Package `{}` does not export `{}`", pkgName, *fullConstant);
```

Comment:
Is it worth having an autocorrect here?

Reply:
or hm it looks like maybe the autocorrect is added earlier?

Reply:
We already get an autocorrect here. We actually add it a few lines up, here:

https://github.com/sorbet/sorbet/blob/60e7ec833ceb2ef8721d64149446fbd9003f1648/resolver/SuggestPackage.cc#L323-L325